packages/core/src/GlobalStyles/GlobalStyles.tsx

```diff
@@ -6,6 +6,9 @@ import tokens from '@contentful/f36-tokens';
 export const GlobalStyles = ({
   withNormalize = true,
 }: {
+  /**
+   * Determines if the browser common style should be normalized
+   */
```

Comment:
Your description might sound like it's our custom internal thing. I would make it as clear as possible that it's just commonly used `normalise.css.` So if the users already have one in their project, it's clear for them that they need to pass `false` here.  WDYT?
```suggestion
  /**
   * Determines if normalise.css should be added, [more info](https://github.com/necolas/normalize.css)
   * 
   * @default true
   */
```